File: Geek Assistant (Android AIO)/Modules/General/Function/GA_Ver.vb
Module GA_Ver
    'Dim Gaver As String
    Public Function Run(level As String) As String
        Dim cDateByNHKomaiha As String = "©2021 By NHKomaiha"
        Dim result As String = $"v{Version.Major}.{Version.Minor}"

        Select Case Version.Revision
            'Case 0
            '    GAver &= Nothing '(Normal)
            Case 1
                result &= " #Beta"
            Case 2
                result &= " #Test"
            Case 3
                result &= " #Dev"
        End Select
        Select Case level
            Case "log"
                result = $"Geek Assistant {result} {cDateByNHKomaiha}."
            Case "Main"
                result &= $" {cDateByNHKomaiha}."
            Case "MainTitle"
                result = $"Geek Assistant — {result}"
            Case "ToU"
                result = $"{My.Application.Info.Copyright}. {result}"
        End Select

        Return result
    End Function
End Module

File: Geek Assistant (Android AIO)/Modules/General/Function/GA_Ver.cs

using System.Diagnostics;
using System.Reflection;
using System.Windows.Forms;

internal static partial class GA_Ver {
    /// <summary>
    /// Create a customized version string accordingly or vX.x if not specified
    /// </summary>
    /// <param name="level">Place where this is being called at</param>
    /// <returns>
    /// <list type="bullet">
    /// <item>log = Geek Assistant vX.x #Phase ©20XX By NHKomaiha.</item>
    /// <item>Main = vX.x #Phase ©20XX By NHKomaiha.</item>
    /// <item>MainTitle = Geek Assistant — vX.x #Phase</item>
    /// <item>ToU = ©2021 Geek Assistant By NHKomaiha. vX.x #Phase"</item>
    /// <item>(Other) = vX.x</item>
    /// </list>
    /// </returns>
    public static string Run(string level = null) {
        string cDateByNHKomaiha = "©2021 By NHKomaiha";
        string result = $"v{c.V.Major}.{c.V.Minor}";
        switch (c.V.Revision) { 
        case 1: {
            result += " #Beta";
            break; }
        case 2: {
            result += " #Test";
            break; }
        case 3: {
            result += " #Dev";
            break; }
        }

        switch (level) {
        case "log": {
            result = $"Geek Assistant {result} {cDateByNHKomaiha}.";
            break; } 
        case "Main": {
            result += $" {cDateByNHKomaiha}.";
            break; } 
        case "MainTitle": {
            result = $"Geek Assistant — {result}";
            break; } 
        case "ToU": { 
            result = $"{c.C}. {result}";
            break; } 
        }

        return result;
    }
}